### PR TITLE
Update example yaml without external dependency

### PR DESCRIPTION
**Pull request by ScorpioCPH**
FYI, @DjangoPeng @gaocegege-caicloud PTAL.

### CODE REVIEW

**Comment by ScorpioCPH** [comment]
Fix part 4 of https://github.com/caicloud/kubeflow-clientset/issues/18

File: examples/tf-jobs/training/Dockerfile.local.training [hunk 1]
```diff
@@ -0,0 +1,8 @@
+
+FROM cargo.caicloud.io/tensorflow/tensorflow:1.2.1
```
Comment (DjangoPeng) [inline examples/tf-jobs/training/Dockerfile.local.training]: I think @zjx-caicloud would open a PR to push the Dockerfile files of TensorFlow base images to caicloud/dockerfile repo. After merging that, we can pull the images from cargo.caicloud.io registry.

**Comment by DjangoPeng** [comment]
It's necessary for us to build some images for example to get rid of the external dependency. @zjx-caicloud

**Comment by DjangoPeng** [comment]
@zjx-caicloud I'll open a PR on caicloud/dockerfile repo to add kubeflow/tensorflow base images.

**Pull request closed without merging**